csi.proto

```diff
@@ -1257,6 +1323,53 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+message VolumeHealth {
+  option (alpha_message) = true;
+
+  enum Condition {
+    // Volume health condition is unknown; treat it as healthy.
+    UNKNOWN = 0;
+    // Volume is accessible and no problems are detected. 
+    HEALTHY = 1;
+    // Volume is not accessible currently, but there's no reason to
+    // expect data loss. Examples: power outage, network outage,
+    // maintenance.
+    TEMPORARY_INACCESSIBLE = 2;
+    // Volume is accessible currently, but is degraded. Data loss
+    // is unlikely as the storage system is fixing the problem
+    // automatically.
+    TEMPORARY_DEGRADED = 3;
+    // Volume is accessible currently, but a problem is detected that
```

Comment:
Just for my understanding, something like loss of a replica or drive in a raid 5 could be:

- temporary degraded: if you already have a spare configured and the system is currently resyncing/reconstructing
- failure likely: if the disk needs to be manually replaced by a human

Reply:
LGTM

Reply:
Sure, I'll add that to clarify.